### DISCUSSION TITLE
Information on type of Github Logs for ingestion

### DISCUSSION THREAD

**Discussion by akm1501** [community/community#131855] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi Team,

I am looking for ingestion of Github logs into my test SIEM platform. I need the github logs which contain information about Kubernetes secrets, dockercfg and dockerconfigjson.

i am unable to understand and find information about which type of github logs will have these information.

Kindly let me know

Thanks

**Comment by MarcusRTM** [comment]
Hey mate!

**GitHub Logs**:
   - GitHub logs contain valuable data related to code repositories, user activity, and security events.
   - Specifically, you're interested in secrets management.

**Types of GitHub Logs**:
   - You'll want to focus on the following secret-related logs:
     - **kubernetes.io/dockercfg**: Stores serialized Docker credentials in the legacy `~/.dockercfg` format. Used for accessing Docker registries.
     - **kubernetes.io/dockerconfigjson**: An improvement over `.dockercfg`. Features a `.dockerconfigjson` key for securely managing Docker registry credentials.

**Ingesting GitHub Logs into Your SIEM**:
   - Follow these steps:
     - Ensure you have a GitHub Enterprise license.
     - Generate an API key with read permissions in GitHub.
     - Configure access via OAuth2 Authorization Flow or a Personal Access Token.
     - Set up GitHub to push logs into your SIEM using the API key.
     - Enable relevant detection rules related to GitHub events in your SIEM.

**Why Monitor GitHub?**:
   - Your code is your organization's "crown jewel."
   - Monitoring GitHub helps prevent accidental exposure of sensitive information (like secrets) and detects potential threats.

Hope these can help.

**Reply by akm1501** [reply]
@MarcusRTM thank for the information
